docs/details/commands/pmm-agent.md

```diff
@@ -46,7 +46,8 @@ Most options can be set via environment variables (shown in parentheses).
 | `--machine-id=machine-id`              | `PMM_AGENT_SETUP_MACHINE_ID`        | Node machine ID (default is auto-detected).
 | `--metrics-mode=auto`                  | `PMM_AGENT_SETUP_METRICS_MODE`      | Metrics flow mode for agents node-exporter. Can be `push` (agent will push metrics), `pull` (server scrapes metrics from agent) or `auto` (chosen by server).
 | `--node-model=NODE-MODEL`              | `PMM_AGENT_SETUP_NODE_MODEL`        | Node model.
-| `--paths-exporters_base=PATH`          | `PMM_AGENT_PATHS_EXPORTERS_BASE`    | Base path for exporters to use.
+| `--paths-base=PATH`                    | `PMM_AGENT_PATHS_BASE`              | Base path for PMM client, where all binaries, tools and collectors are located. If not set, default is `/usr/local/percona/pmm2`.
```

Comment:
@denisok @JiriCtvrtka I rewrote the path comments to be inline, within the table. For even more clarification, we can add some examples where paths-base/paths-exporters_base is/is not given, and the resulting paths for each.

One other comment. It's probably asking for trouble to use mixed dash and underscore within a parameter name. They are easy to mistype or misread. Is it too late to standardise on full dashes, as all the other option names seem to be?

Reply:
unfortunately those are old ones, we can't rename it, ppl probably already use them in production. We can add new and have them in documentation and leave old ones in the code, so both works. But it is too late now.